classes/Commands/CheckRequirementsCommand.php

```diff
@@ -141,12 +141,49 @@ private function processRequirementErrors(): void
     private function processRequirementWarnings(): void
     {
         foreach ($this->upgradeSelfCheck->getWarnings() as $warning => $value) {
-            $wording = $this->upgradeSelfCheck->getRequirementWording($warning);
-            $this->writeWarning($wording['message']);
-            if (isset($wording['list'])) {
-                foreach ($wording['list'] as $item) {
-                    $this->output->writeln('    ' . $item);
-                }
+            switch ($warning) {
+                case UpgradeSelfCheck::CORE_TEMPERED_FILES_LIST_NOT_EMPTY:
+                    $wording = $this->upgradeSelfCheck->getRequirementWording($warning);
+                    $this->writeWarning($wording['message']);
+                    $missingFiles = $this->upgradeSelfCheck->getCoreMissingFiles();
+
+                    $this->output->writeln('    ' . count($missingFiles) . ' files are missing:');
+                    foreach ($missingFiles as $missingFile) {
+                        $this->output->writeln('        ' . $missingFile);
+                    }
+
+                    $alteredFiles = $this->upgradeSelfCheck->getCoreAlteredFiles();
+
+                    $this->output->writeln('    ' . count($alteredFiles) . ' files are altered:');
+                    foreach ($alteredFiles as $alteredFile) {
+                        $this->output->writeln('        ' . $alteredFile);
+                    }
+                    break;
+                case UpgradeSelfCheck::THEME_TEMPERED_FILES_LIST_NOT_EMPTY:
+                    $wording = $this->upgradeSelfCheck->getRequirementWording($warning);
+                    $this->writeWarning($wording['message']);
+                    $missingFiles = $this->upgradeSelfCheck->getThemeMissingFiles();
+
+                    $this->output->writeln('    ' . count($missingFiles) . ' files are missing:');
+                    foreach ($missingFiles as $missingFile) {
+                        $this->output->writeln('        ' . $missingFile);
+                    }
+
+                    $alteredFiles = $this->upgradeSelfCheck->getThemeAlteredFiles();
+
+                    $this->output->writeln('    ' . count($alteredFiles) . ' files are altered:');
+                    foreach ($alteredFiles as $alteredFile) {
+                        $this->output->writeln('        ' . $alteredFile);
+                    }
+                    break;
+                default:
+                    $wording = $this->upgradeSelfCheck->getRequirementWording($warning);
+                    $this->writeWarning($wording['message']);
```

Comment:
I see these 2 lines are always the same. Shall we move them before the switch to have the common behavior for all cases?

Reply:
for me "See the list of alterations." nothing to do on the CLI version, this clearly refers to a button allowing you to display a modal